README.md

```diff
@@ -7,9 +7,10 @@ An open source Python based app that replicates the functionality of the existin
 
 **Important Note:** tabcmd is a work in progress ("beta") which may be useful for test and development purposes, but is not yet recommended for production environments.
 
-
+tabcmd.exe is available at ____
 ## For user documentation see https://tableau.github.io/tabcmd/
```

Comment:
When I hit the download button on https://tableau.github.io/tabcmd/, it takes me to  https://codeload.github.com/tableau/tabcmd/zip/2.0.0 and I get a 404.